From aef95e59e918ed9aaaf7e5c2daf32ff205ed314c Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.de>
Date: Sat, 17 Sep 2022 00:40:49 +0200
Subject: [PATCH] fix(nvmf): don't create did-setup file

did-setup files are meant to indicate that an interface setup
was successful. Don't do it here.

Resolves: #1933787
---
 modules.d/95nvmf/parse-nvmf-boot-connections.sh | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/modules.d/95nvmf/parse-nvmf-boot-connections.sh b/modules.d/95nvmf/parse-nvmf-boot-connections.sh
index cfed5ae5..1cdaba69 100755
--- a/modules.d/95nvmf/parse-nvmf-boot-connections.sh
+++ b/modules.d/95nvmf/parse-nvmf-boot-connections.sh
@@ -102,9 +102,6 @@ done
 
 if [ -f "/etc/nvme/discovery.conf" ]; then
     /sbin/initqueue --settled --onetime --unique --name nvme-discover /usr/sbin/nvme connect-all
-    if [ "$trtype" = "tcp" ]; then
-        : > /tmp/net."$ifname".did-setup
-    fi
 else
     # No nvme command line arguments present, try autodiscovery
     if [ "$trtype" = "fc" ]; then

